src/main/java/com/example/bookstoreapp/controller/BookController.java

```diff
@@ -70,9 +70,17 @@ public BookDto updateBook(@PathVariable Long id, @RequestBody BookRequestDto boo
     }
 
     @ResponseStatus(HttpStatus.NO_CONTENT)
-    @PreAuthorize("hasAnyRole('USER','ADMIN')")
+    @PreAuthorize("hasRole('ADMIN')")
     @DeleteMapping("/{id}")
     public void delete(@PathVariable Long id) {
         bookService.deleteById(id);
     }
+
+    @GetMapping("/category/{categoryId}")
```

Comment:
Looks like wrong name of enpoint
![image](https://github.com/fedaktaras/book-store-app/assets/69054906/6bdc0bd0-4fd5-4135-9e73-806105462ea3)

Reply:
I have this in CategoryCintroller